packages/jsii-rosetta/test/translations/expressions/double_quoted_dict_keys.java

```diff
@@ -1 +1 @@
-Map<String, String> x = Map.of("key", "value");
+final Map<String, String> x = Map.of("key", "value");
```

Comment:
I'm not sure that I agree with this change.

[Yes, it's technically more correct to translate `const` to `final`.](https://user-images.githubusercontent.com/524162/98555497-4bff4900-22a2-11eb-9fa3-59706593f796.png)


But, the goal of this tool is not to produce perfect translations -- the goal is to produce acceptable sample code, and the primary purpose of example code is that it is easy to read.

I find the additional `final` to be distracting, when what we're really trying to get consumers to understand is that they're declaring a variable.

Reply:
Yeah you've got a point right here....